adrdev10/autoseers
# Adds firebase analytics boilerplate code

#### Description
Adds the initial set up for implementing firebase analytics

### What
This PR makes the following changes:
- Adds DI code to get firebase analytics instance
- Creates a new module for injecting the AnalyticsService to the onboarding viewmodel
- Creates firebase events to log name, phone, code, and user creation events (Currently, the only event that's being logged is the phone event `add_phone_number` but will the others tomorrow)

#### Ways to test
- Run app 
- Get to the phone screen
- Enter phone number
- Press the continue button
- Navigate to logcat
- Verify that you see this event: `Logging event: origin=app,name=add_phone_number,params=Bundle[{ga_event_origin(_o)=app, ga_screen_class(_sc)=MainActivity, ga_screen_id(_si)=-4659317883940877624}]`
